### ISSUE TITLE
see coverage of branches and pull requests in the dashboard

### ISSUE THREAD

**Issue by 304NotModified** [codecov/support#208]
Before it was possible so easily see the coverage of the branches and pull requests.

But now those are not listed in the overviews:

https://codecov.io/gh/NLog/NLog/branches
![image](https://cloud.githubusercontent.com/assets/5808377/15364264/28721a84-1d1c-11e6-9743-9dfb718195f0.png)

https://codecov.io/gh/NLog/NLog/pulls
![image](https://cloud.githubusercontent.com/assets/5808377/15364269/2ebfa9ec-1d1c-11e6-9bb5-a2e29f073f5f.png)

Is it possible to show the coverage for every branch and pull request in those pages?

**Comment by stevepeak** [comment]
Certainly! Thank you for that suggestion. I'll add some coverage data to these pages.

Thanks!
